airbyte-integrations/connectors/source-faker/acceptance-test-config.yml

```diff
@@ -14,7 +14,7 @@ acceptance_tests:
     tests:
       - config_path: secrets/config.json
         backward_compatibility_tests_config:
-          disable_for_version: "0.2.1"
+          disable_for_version: "1.0.0"
```

Comment:
```suggestion
          disable_for_version: "1.0.0" # We changed the cursor field of the Purchases stream in 2.0.0
```
I think we shall introduce a `reason` field under the `disable_for_version`, in the same philosophy as `bypass_reason`